search.py

```diff
@@ -109,8 +121,29 @@ def cosine_score(tokens_arr):
             continue
 
         # 2. Obtain the second vector's (query vector's) value for pointwise multiplication
-        # Calculate the weight entry of the term in the query, of the term-document matrix
-        query_term_weight = get_query_weight(posting_list.size, term_frequencies[term])
+
+        query_term_weight = get_query_weight(posting_list.size, term_frequencies[term]) # without Rocchio
+
+        if (True):
```

Comment:
typo?

Reply:
Nope, it's just currently forced to always apply Rocchio. Will apply the identification of search type later.